docs/04-testing.md

```diff
@@ -62,7 +54,13 @@ Then, copy the following template to use an in-memory sqlite database and enable
     <env name="DB_CONNECTION" value="testing"/>
     <env name="APP_KEY" value="base64:2fl+Ktvkfl+Fuz4Qp/A75G2RTiWVA/ZoKZvp6fiiM10="/>
   </php>
+  <source>
```

Comment:
I am getting erros on my package for this source element to not exist, @hans-thomas can you show it working locally for you on the PHPUnit 10 version?

Reply:
[see this](https://github.com/hans-thomas/sphinx/blob/565120a90dbbd5c4c0450d5e402a65cb7a1b3284/phpunit.xml#L19C1-L23C12). please make sure that you're using `phpunit:^10`.